### ISSUE TITLE
dos2unix: command not found on Continuous Deployment

### ISSUE THREAD

**Issue by eloekset** [Microsoft/PartsUnlimitedMRP#94]
The **SSH-MRP-Artifacts.ps1** fails when executing the **plink.txt** file with the following error message:
*bash: line 2: dos2unix: command not found*

This is my log file for the **Deploy MPR using SFTP and SSH through PowerShell** step:
[3_Deploy MPR using SFTP and SSH through PowerShell.txt](https://github.com/Microsoft/PartsUnlimitedMRP/files/522991/3_Deploy.MPR.using.SFTP.and.SSH.through.PowerShell.txt)

Compared to the log file in issue #75, I see that in my log there's an issue running sudo:
*Store key in cache? (y/n) sudo: no tty present and no askpass program specified*

I'm not sure if that's a real problem, but obviously the apt-get command to install dos2unix hasn't run successfully.

**Comment by julienstroheker** [comment]
Hi, 

I just did the test on my side and everything works fine. 

Some questions :
* Did you deploy the target VM using the template in the HOL [here](https://microsoft.github.io/PartsUnlimitedMRP/fundvsts/fund-02-MS-CD.html) 
* I saw your named your build definition : PartsUnlimitedMRP.CI but how did you name your release? 

Thanks

**Comment by julienstroheker** [comment]
@eloekset any news ?
Thanks

**Comment by eloekset** [comment]
Sorry, I haven't had time to do another test yet. I did deploy the target VM using the HOL template, but that was a few days ago, so I haven't got the latest changes if there are any. I hope to find some time to test it later these evening, if not, it won't be until some time next week.

**Comment by julienstroheker** [comment]
NP,

Let me know.

Thanks

**Comment by eloekset** [comment]
I re-deployed the HOL template and now there's no askpass issue. The release worked fine, and I confirmed that the site is up at ...:9080/mrp. 
![2016-10-13_21-23-44](https://cloud.githubusercontent.com/assets/791721/19363858/185029a4-918c-11e6-89d1-b3a4b0ed5549.png)

The best thing is, I've been wondering for a while how to run remote .sh scripts using the SSH task, and I think the dos2unix tool will fix the issue that I had there - otherwise unrelated to this issue. But I've learned a lot from your tutorial so far, so thanks for building these labs, and thanks for your help!
